language101_scraper.py

```diff
@@ -12,20 +13,20 @@
 from bs4 import BeautifulSoup
 
 
+
 parser = argparse.ArgumentParser(description='Scrape full language courses by Innovative Language.')
 parser.add_argument('-u', '--username', help='Username (email)')
 parser.add_argument('-p', '--password', help='Password for the course')
 parser.add_argument('--url', help='URL for the first lesson of the course')
 
 args = parser.parse_args()
 
-
 USERNAME = args.username or input('Username(mail): ')
 PASSWORD = args.password or input('Password: ')
 COURSE_URL = args.url or input('Please insert first lesson URL of the desired course, for example:\n'
-    ' * https://www.japanesepod101.com/lesson/lower-beginner-1-a-formal-japanese-introduction/?lp=116\n'
-    ' * https://www.spanishpod101.com/lesson/basic-bootcamp-1-a-pleasure-to-meet-you/?lp=425\n'
-    ' * https://www.chineseclass101.com/lesson/absolute-beginner-1-meeting-whats-your-name/?lp=208\n')
+                               ' * https://www.japanesepod101.com/lesson/lower-beginner-1-a-formal-japanese-introduction/?lp=116\n'
+                               ' * https://www.spanishpod101.com/lesson/basic-bootcamp-1-a-pleasure-to-meet-you/?lp=425\n'
+                               ' * https://www.chineseclass101.com/lesson/absolute-beginner-1-meeting-whats-your-name/?lp=208\n')
```

Comment:
```suggestion
    ' * https://www.japanesepod101.com/lesson/lower-beginner-1-a-formal-japanese-introduction/?lp=116\n'
    ' * https://www.spanishpod101.com/lesson/basic-bootcamp-1-a-pleasure-to-meet-you/?lp=425\n'
    ' * https://www.chineseclass101.com/lesson/absolute-beginner-1-meeting-whats-your-name/?lp=208\n')
```
Why do you need these extra spaces?